src/main/java/org/gridsuite/loadflow/server/service/LoadflowObserver.java

```diff
@@ -7,63 +7,26 @@
 package org.gridsuite.loadflow.server.service;
 
 import com.powsybl.loadflow.LoadFlowResult;
-import io.micrometer.core.instrument.Counter;
 import io.micrometer.core.instrument.MeterRegistry;
-import io.micrometer.observation.Observation;
 import io.micrometer.observation.ObservationRegistry;
 import lombok.NonNull;
+import org.gridsuite.loadflow.server.dto.LoadFlowParametersInfos;
+import org.gridsuite.loadflow.server.service.computation.AbstractObserver;
 import org.springframework.stereotype.Service;
 
+import static org.gridsuite.loadflow.server.service.LoadFlowWorkerService.COMPUTATION_LABEL;
```

Comment:
Rename by COMPUTATION_TYPE

Reply:
I don't agree. This is not a type (which would define a computation type, through an enum or whatever). This really is a string label. 

But it is the LOADFLOW_LABEL, not a generic COMPUTATION_LABEL. So I'd rather rename it like this.